boards/arm/nucleo_f401re/Kconfig.defconfig

```diff
@@ -44,4 +45,14 @@ config PWM_STM32_2
 
 endif # PWM
 
+if WATCHDOG
+
+config WWDG_STM32
+	default y
+
+config IWDG_STM32
```

Comment:
Yes, because as you can [here](https://github.com/zephyrproject-rtos/zephyr/blob/master/soc/arm/st_stm32/common/Kconfig.defconfig.series#L44) under the soc/ `IWDG_STM32` is enabled by default. In order to not break other things that use it I haven't change it.